### PR TITLE
feat: added a toggle between dark and light mode

### PR DESCRIPTION
**Pull request by PrathameshSonje**
# Related Issue

Closes: #90 

## Describe the changes you've made

Instead of a dropdown menu added a toggle functionality to switch between dark and light mode

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -5,8 +5,8 @@ NEXT_PUBLIC_CLERK_SIGN_UP_URL=/join
 NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL=/dashboard
 NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL=/welcome
 
-NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=
-CLERK_SECRET_KEY=
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_cmVuZXdpbmctdGV0cmEtOTguY2xlcmsuYWNjb3VudHMuZGV2JA
```
Comment (NewtonFernandis) [inline .env.example:8]: Dont expose these keys, Please add this file in .gitIgnore

**Review by NewtonFernandis — CHANGES_REQUESTED** [review]
Dont expose these keys, Please add this file in .gitIgnore

File: .env.example (hunk 1 above)
Comment (PrathameshSonje) [reply .env.example]: I have updated the .env.example file.

**Comment by samyabrata-maji** [comment]
looks good to me

**Review by samyabrata-maji — APPROVED** [review]

**Pull request merged**